### PR TITLE
[v7r0] MP tests and fixes

### PR DESCRIPTION
**Pull request by fstagni**
User jobs might need to know how many processors are available. If a WN is running several MP (or SP, or mix of MP and SP) jobs in parallel, then the info on the available processors must be per-job. 

This goes together with https://github.com/DIRACGrid/Pilot/pull/98

BEGINRELEASENOTES

*Subsystem
CHANGE: save the number of processors as jobLimits in pilot cfg

ENDRELEASENOTES

### CODE REVIEW

File: WorkloadManagementSystem/Agent/JobAgent.py [hunk 1]
```diff
@@ -292,7 +295,7 @@ def execute(self):
       self.log.warn('Job has no CPU requirement defined in JDL parameters')
 
     # Job requirement for a number of processors
-    processors = int(params.get('NumberOfProcessors', 1))
+    processors = int(params.get('NumberOfJobProcessors', 1))
```
Comment (atsareg) [inline WorkloadManagementSystem/Agent/JobAgent.py]: May be this option should be called JobProcessorsRequirement in order not to confuse with a similar name but different meaning NumberOfProcessors option

Comment (atsareg) [inline WorkloadManagementSystem/Agent/JobAgent.py]: This is a job JDL parameter which is NumberOfProcessors, you can not change the name here. The NumberOfJobProcessors that you assign in the Computing Element is not the same thing.

Comment (fstagni) [reply WorkloadManagementSystem/Agent/JobAgent.py]: OK fixed.

**Comment by fstagni** [comment]
Review, please.

**Pull request merged**